### PR TITLE
build: deploy to stagging

### PR DESCRIPTION
**Pull request by alexgarel**
Setting up CI to deploy to stagging (before going for prod)

### CODE REVIEW

**Comment by alexgarel** [comment]
@simonj2 FYI I have elasticvue working: https://elasticvue.search.openfoodfacts.net/
The address for ES is https://elasticvue.search.openfoodfacts.net/es/ 
But you will need the user / password !

**Comment by alexgarel** [comment]
@simonj2 this PR is ready for review and merging :-)

File: .env [hunk 1]
```diff
@@ -12,12 +12,22 @@ STACK_VERSION=8.3.3
 CLUSTER_NAME=docker-cluster
 
 # Set to 'basic' or 'trial' to automatically start the 30-day trial
-LICENSE=basic
+# LICENSE=basic
```
Comment (simonj2) [inline .env:15]: Remove this perhaps?

File: .env [hunk 2]
```diff
@@ -12,12 +12,22 @@ STACK_VERSION=8.3.3
...
-ES_VUE_PORT=127.0.0.1:8080
 SEARCH_PORT=127.0.0.1:8000
+ES_VUE_PORT=127.0.0.1:8080
+# this service needs to be accessible by product opener
+# this means publishing it if we are on another machine
+REDIS_PORT=127.0.0.1:6379
+# exposures for dev
+ES_PORT=127.0.0.1:9200
+
+# eventually add basic auth for nginx
```
Comment (simonj2) [inline .env:26]: This is built, right? So I think this line can be removed.

File: app/scripts/perform_import_parallel.py [hunk 3]
```diff
@@ -131,23 +152,30 @@ def get_redis_updates(next_index):
 
 
 def perform_import(filename, num_items, num_processes, start_time):
+    """Main function running the import sequence
+    """
     es = connection.get_connection()
+    # we create a temporary index to import to
+    # at the end we will change alias to point to it
```
Comment (simonj2) [inline app/scripts/perform_import_parallel.py:159]: Thanks for adding these comments :)

**Review by simonj2 — APPROVED** [review]

File: .env (hunk 2 above)
Comment (alexgarel) [reply .env:26]: no I put eventually, in the sens that if it's empty it does not add it.

File: .github/workflows/container-deploy.yml [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        env:
+          # only stagging for now
+          # Note: env is also the name of the directory on the server
+          - off-search-net
+          # - ${{ startsWith(github.ref, 'refs/tags/v') && 'off-search-org' || 'off-search-net' }}
```
[the remark this answers is not in the document]
Comment (alexgarel) [reply .github/workflows/container-deploy.yml:29]: I keep it because we will use it when we enable prod deployment.

**Pull request merged**